app/models/manageiq/providers/ibm_power_hmc/infra_manager/metrics_capture.rb

```diff
@@ -1,19 +1,18 @@
-class ManageIQ::Providers::IbmPowerHmc::InfraManager::MetricsCapture < ManageIQ::Providers::BaseManager::MetricsCapture
+class ManageIQ::Providers::IbmPowerHmc::InfraManager::MetricsCapture < ManageIQ::Providers::InfraManager::MetricsCapture
   VIM_STYLE_COUNTERS = {
     "cpu_usage_rate_average"     => {
       :counter_key           => "cpu_usage_rate_average",
       :instance              => "",
-      :capture_interval      => "20",
+      :capture_interval      => "30",
```

Comment:
I believe we still have a requirement that datapoints have to be every 20seconds unfortunately, every 30 is annoying because you'll have to aggregate 2 per minute then interpolate to 3 per minute cc @Fryguy

Reply:
ok